website/blog/2024-06-27-new-ferretdb-v122-user-setup-feature.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: new-ferretdb-v122-user-setup-feature
+title: FerretDB releases v1.22 with an initial FerretDB user setup feature
+authors: [alex]
+description: >
+  We have released FerretDB v1.22 with an initial FerretDB user setup feature, configurable document size limit, and several bug fixes and enhancements.
+image: /img/blog/ferretdb-v1.22.0.jpg
+tags: [release]
+---
+
+![FerretDB v1.22](/img/blog/ferretdb-v1.22.0.jpg)
+
+We are thrilled to announce the release of FerretDB v1.22.0, which now includes an initial user setup feature, a configurable document size limit, and several bug fixes and enhancements.
+
+<!--truncate-->
+
+This release, one of the last in the FerretDB v1.x series, adds new features and improvements to make FerretDB even better.
+Users can now set up an initial user for authentication, configure the document size limit, and benefit from several enhancements in this release.
+
+As we continue to build and improve [FerretDB](https://www.ferretdb.com/), we are committed to providing a truly open-source document database that satisfies many MongoDB use cases.
+In the background, we are working on FerretDB v2.0, which will drastically improve performance and compatibility.
+It will also be a complete departure from our current architecture, and we can't wait to share it with you soon.
+
+Read on to learn more about what's new in FerretDB v1.22.0.
+
+## Enable initial FerretDB user setup
+
+A standout feature in FerretDB v1.22 is the ability to set up an initial user for authentication.
+This makes it easier for users to configure their FerretDB instance securely right from the start using dedicated flags or environment variables.
+Here's how you can do it:
+
+```sh
+ferretdb --test-enable-new-auth=true --setup-username=user --setup-password=pass --setup-database=ferretdb
+```
+
+- `--setup-username`/`FERRETDB_SETUP_USERNAME`: Specifies the username to be created.
+- `--setup-password`/`FERRETDB_SETUP_PASSWORD`: Specifies the password for the user (can be empty).
+- `--setup-database`/`FERRETDB_SETUP_DATABASE`: Specifies the initial database that will be created.
+- `--test-enable-new-auth`/`FERRETDB_TEST_ENABLE_NEW_AUTH`: Must be set to `true` to enable the new authentication setup.
+
+Once the flags/environment variables are passed, FerretDB will create the specified user with the given password and database.
```

Comment:
It does not create a user with a given [...] database

Reply:
So something like this: `Once the flags/environment variables are passed, FerretDB will create the specified user with the given password and the given database.`?